packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -40,62 +100,225 @@ export class SankeyChartBase extends React.Component<
       theme: theme!,
       width: this.state.containerWidth,
       height: this.state.containerHeight,
-      pathColor: pathColor,
+      pathColor,
       className,
     });
-    const margin = { top: 10, right: 0, bottom: 10, left: 0 };
-    const width = this.state.containerWidth - margin.left - margin.right;
-    const height =
-      this.state.containerHeight - margin.top - margin.bottom > 0
-        ? this.state.containerHeight - margin.top - margin.bottom
-        : 0;
+    const margin = { top: 36, right: 48, bottom: 32, left: 48 };
+    const width = this.state.containerWidth - margin.right;
+    const height = this.state.containerHeight - margin.bottom > 0 ? this.state.containerHeight - margin.bottom : 0;
 
     const sankey = d3Sankey
       .sankey()
-      .nodeWidth(5)
-      .nodePadding(6)
+      .nodeWidth(124)
       .extent([
-        [1, 1],
+        [margin.left, margin.top],
         [width - 1, height - 6],
       ]);
 
+    const nodesInColumn: NodesInColumns = this._populateNodeInColumns(sankey);
+    if (!this.state.appliedHeightChange) {
+      this._adjustOnePercentHeightNodes(sankey, nodesInColumn);
+      this._nodePadding = this._adjustPadding(sankey, height - 6, nodesInColumn);
+      this.setState({
+        appliedHeightChange: true,
+      });
+    }
+    sankey.nodePadding(this._nodePadding);
     sankey(this.props.data.SankeyChartData!);
+    this._populateNodeActualValue(this.props.data.SankeyChartData!);
+    this._assignNodeColors();
     const nodeData = this._createNodes(width);
     const linkData = this._createLinks();
     return (
-      <div
-        className={this._classNames.root}
-        role={'presentation'}
-        ref={(rootElem: HTMLDivElement) => (this.chartContainer = rootElem)}
-      >
-        <svg width={width} height={height} id={getId('sankeyChart')}>
-          <g className={this._classNames.nodes}>{nodeData}</g>
-          <g className={this._classNames.links} strokeOpacity={0.2}>
-            {linkData}
-          </g>
-        </svg>
-      </div>
+      <>
+        <div id="tooltip" className={this._classNames.toolTip} style={{ position: 'absolute', display: 'none' }} />
```

Comment:
What does this tooltip represent

Reply:
This tooltip is there for showing the truncated value for node name.